src/main/java/com/SpringBoot/SpringBootDatabaseProject/repository/UserRepository.java

```diff
@@ -4,5 +4,4 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface UserRepository extends JpaRepository<User, Integer> {
-    User findByName(String name);
```

Comment:
Bunu silme nededin ne? (Ogrenmek icin soruyorum)

Reply:
İlk başta yapmış olduğum service katmanında:    
public Product getProductByName(@PathVariable String name){
        return productService.getProductsByName(name);
bu şekilde bir kod parçası vardı. Ancak findByName methodu repository interface ' inde tanımlı olmadığı için yazmıştım.Daha sonra kod parçası üzerinde exception sınıfı eklerken bazı methodları kaldırdığım için (exception kavramını daha basit anlayabilmek için)  getProductByName methodunu kaldırdım ve interface sınıfından kullanmadığı için kaldırdım.

Reply:
Ihtiyac olmadigi icin kaldirdin, ok.

Reply:
Mesela, `List` kullanacagin zaman gidip `import.java.util.*;` yerine `import java.util.List;` dersin.